src/server/cray/cfs/api/controllers/sessions.py

```diff
@@ -549,6 +558,42 @@ def _validate_session_target(target):
     return None
 
 
+class ArgumentParserError(Exception):
+    pass
+
+
+class ThrowingArgumentParser(argparse.ArgumentParser):
+    def error(self, message):
+        raise ArgumentParserError(message)
+
+
+def _validate_ansible_passthrough(passthrough):
+    """Validate the ansible_passthrough
+
+    :param passthrough: Config Framework Session Ansible passthrough
+    :type passthrough: string
+
+    :rtype: None or connexion.problem if errors occur
+    """
+
+    parser = ThrowingArgumentParser()
+    parser.add_argument('-e', '--extra-vars', type=str)
+    parser.add_argument('-f', '--forks', type=int)
+    parser.add_argument('--skip-tags', type=str)
+    parser.add_argument('--start-at-task', type=str)
+    parser.add_argument('-t', '--tags', type=str)
+    passthrough_arguments = shlex.split(passthrough, posix=False)
+    try:
+        parser.parse_args(passthrough_arguments)
+    except Exception as e:
+        return connexion.problem(
+            detail="Error validating ansible-passthrough: {}".format(e),  # noqa: E501
```

Comment:
Should this be `"Error validating --ansible-passthrough..."`?

Reply:
I didn't use the format because this is returned by any api call, not just the cli.